package.json

```diff
@@ -17,6 +17,7 @@
   "workspaces": {
     "packages": [
       "apps/*",
+      "expokit-npm-package",
```

Comment:
Could this package live under "packages/expokit"? (If so, would like to move it in another PR)

Reply:
yeah, I was thinking that as well. Will just take a bit of work to migrate (I think there are still scripts in universe that depend on this location).